example/00-config.yaml

```diff
@@ -4,6 +4,7 @@ kind: Configuration
 issuerName: gardener
 restrictIssuer: true
 acme:
+# noDeactivateAuthorizations: true
```

Comment:
Can we call it `deactivateAuthorizations` and default it to `true` instead of taking the inverted version?

Reply:
That's funny. I started with your suggested variant, but thought it would be better to have the default to `false`.
But probably you are right, it is a little bit confusing.
Done.